infra/elasticsearch/variables.tf

```diff
@@ -0,0 +1,25 @@
+terraform {
+  required_version = "~> 0.11.0"
+
+  backend "s3" {
+    bucket     = "terraform.analytics.justice.gov.uk"
+    key        = "elasticsearch/terraform.tfstate"
+    region     = "eu-west-1"
+    encrypt    = true
+    kms_key_id = "arn:aws:kms:eu-west-1:593291632749:key/df8888e3-4080-4c2b-a71e-1425e72f98e4"
```

Comment:
Where is this coming from?

Reply:
KMS

Reply:
I should have been more specific 🤣 - what I meant was, "who/what created this key?"
Maybe I missed the relevant terraform bits.